src/frontend/screens/Settings/sections/SystemInfo/vendorLogo.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react'
+
+import { ReactComponent as AMDLogo } from 'frontend/assets/amd-logo.svg'
+import { ReactComponent as NVIDIALogo } from 'frontend/assets/nvidia-logo.svg'
+import { ReactComponent as IntelLogo } from 'frontend/assets/intel-logo.svg'
+import { ReactComponent as AppleLogo } from 'frontend/assets/apple-logo.svg'
+
+interface Props {
+  model?: string
+}
+
+function VendorLogo({ model }: Props) {
+  if (model?.toLowerCase().includes('nvidia'))
```

Comment:
```suggestion
  if (!model) {
    return <></>
  }

  model = model.lowerCase()

  if (model.includes('nvidia'))
```

Just a suggestion, I think it's simpler to read and slightly faster if we return early if `model` is undefined.